<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoWall</class>
 <widget class="QWidget" name="PhotoWall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>255</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>255</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: transparent;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="2" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="pictureView" name="graphicsView">
       <property name="styleSheet">
        <string notr="true">background-color: transparent;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>280</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnL">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 85, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="pictureButton" name="btnPic0" native="true"/>
       </item>
       <item>
        <widget class="pictureButton" name="btnPic1" native="true"/>
       </item>
       <item>
        <widget class="pictureButton" name="btnPic2" native="true"/>
       </item>
       <item>
        <widget class="pictureButton" name="btnPic3" native="true"/>
       </item>
       <item>
        <widget class="pictureButton" name="btnPic4" native="true"/>
       </item>
       <item>
        <widget class="pictureButton" name="btnPic5" native="true"/>
       </item>
       <item>
        <widget class="pictureButton" name="btnPic6" native="true"/>
       </item>
       <item>
        <widget class="pictureButton" name="btnPic7" native="true"/>
       </item>
       <item>
        <widget class="pictureButton" name="btnPic8" native="true"/>
       </item>
       <item>
        <widget class="pictureButton" name="btnPic9" native="true"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnR">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 85, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>280</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pictureView</class>
   <extends>QGraphicsView</extends>
   <header>pictureview.h</header>
  </customwidget>
  <customwidget>
   <class>pictureButton</class>
   <extends>QWidget</extends>
   <header>picturebutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
